openlibrary/plugins/upstream/mybooks.py

```diff
@@ -19,7 +19,6 @@
 from openlibrary.core.models import LoggedBooksData
 from openlibrary.core.yearly_reading_goals import YearlyReadingGoals
 
-
```

Comment:
Maybe drop these if you can since they're in an unrelated file.

Reply:
Yeah, this is a relic of my old struggles with precommits. The unnecessary changes have been removed, and should not show up in the future.